openadapt/record.py

```diff
@@ -628,11 +628,44 @@ def record(
 
     # TODO: discard events until everything is ready
 
+    # Variables for sequence detection
+    sequence = ['o', 'a', '.', 's', 't', 'o', 'p']
+    current_index = 0
```

Comment:
What do you think about naming this more descriptively, e.g. `stop_sequence_index`.

In that case we may also need to maintain multiple indexes, one for each sequence in `config.STOP_SEQUENCES`.